backend/routes/service.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const { models } = require("../sequelize");
+const extractFiles = require("../middlewares/files");
+const files = require("../middlewares/files");
+
+const router = express.Router();
+
+// {TODO: add exception handling}

```

Comment:
nit: add your username so it's clear who owns this todo. Format: 
TODO(<username>): add exception handling